rca/users/birdbath.py

```diff
@@ -0,0 +1,98 @@
+from birdbath.processors import BaseProcessor
+from django.conf import settings
+from faker import Faker
+
+from rca.people.models import StudentPage
+from rca.users.models import User
+
+
+class BaseUserAnonymiser(BaseProcessor):
+    def get_queryset(self):
+        users = User.objects.all()
+
+        if settings.BIRDBATH_USER_ANONYMISER_EXCLUDE_SUPERUSERS:
+            users = users.exclude(is_superuser=True)
+
+        if settings.BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE:
+            users = users.exclude(
+                email__regex=settings.BIRDBATH_USER_ANONYMISER_EXCLUDE_EMAIL_RE
+            )
+
+        # Exclude users with an email that matches (end with) @rca.ac.uk
+        # Users are all students as they use email addresses ending with @network.rca.ac.uk
+        users = users.exclude(email__endswith="@rca.ac.uk")
+
+        return users
+
+
+class StudentAccountAnonymiser(BaseUserAnonymiser):
```

Comment:
I noticed that, after pulling dev data and running birdbath, in the list of Users you see the anonymised name on the left but can see the original name as a student group on the right - Where is says Student:
![Screenshot 2024-02-05 at 16 56 01](https://github.com/torchbox/rca-wagtail-2019/assets/117450473/f39b6d3a-b6f1-494d-ae1b-93e05cdfd90e)

Reply:
I think this is acceptable given the account name is treated.